x-pack/legacy/plugins/apm/server/lib/traces/get_trace_items.ts

```diff
@@ -52,6 +52,6 @@ export async function getTraceItems(traceId: string, setup: Setup) {
 
   return {
     items: resp.hits.hits.map(hit => hit._source),
-    exceedsMax: resp.hits.total > maxTraceItems
+    exceedsMax: resp.hits.total.value > maxTraceItems
```

Comment:
With the new settings counts above 10.000 are no longer accurate. Perhaps for this specific query we want to know the accurate counts, and should apply `track_total_hits: true`?